src/components/entryComponents/UploadImgButton.tsx

```diff
@@ -12,14 +12,22 @@ const VisuallyHiddenInput = styled("input")`
 `;
 
 export default function UploadImgButton() {
+  const me = useAppSelector(s => s.gameState.me)
   const [userImg, setUserImg] = useState<string>(
-    "https://placehold.co/600x400?text=Choose+your+picture"
-  );
+    'https://placehold.co/600x400/263238/48af63?text=Upload+Image');
```

Comment:
I think we should rely less on third party service to make the app more stable and independent of other sites

Reply:
Ok, changed it to static asset image